Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -98,14 +82,11 @@ static void HotplugThreadFunc()
     // Only react to "device added" events for evdev devices that we can access.
     else if (strcmp(action, "add") == 0 && access(devnode, W_OK) == 0)
```

Comment:
Should we test for R_OK here? It'll end up with the same result, through the constructor -> initialized false, though, just saves the alloc/open. Or is there never going to be any case where there is write but no read access?

Either way, I'm fine with it.

Reply:
I must have been very tired. I didn't actually remove the call to access() like the commit message says. Fixed now.

Reply:
We open the device for reading and writing, so we need to check for neither beforehand.